### DISCUSSION TITLE
Reverse Syntex Is a Bit Slow

### DISCUSSION THREAD

**Discussion by Balrrach** [James-Yu/LaTeX-Workshop#3112] [Q&A]
### Disable all the other extensions except for LaTeX Workshop, restart VS Code, and check that you can not see the requested feature. [Required]

You still see this issue?: Yes

### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before requesting a feature.

You visited the wiki?: Yes

## Problem Description
This is a bit picky but Reverse Syntax is not instantaneous like Direct Syntax it takes halve a second for VS Code to reposition in the right line.

## To Reproduce  [Required]

1. Call Reverse Syntex anywhere on the pdf
2. Wait half a second to be redirected to the code

## Describe the solution you'd like  [Required]
I'd like it to be instantaneous like Direct Syntex is and like it is in the rest of LaTex editors.

### Are you using VSCodium?

No

### Are you using the Snap or Flatpack versions of VS Code?

No

### Are you using LaTeX Workshop with VS Code Remote?

No

### Additional context

I know this is related to the way the extension uses Syntex since this is not the way it works in the rest of the LaTex editors.

**Comment by tamuratak** [comment]
- Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 
- Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

When all of the above is done, we will reopen this issue if needed.
